### PR TITLE
Named tuples

### PR DESCRIPTION
**Pull request by mdcfrancis**
Includes both v3.0.0 and v3.0.1 - only change in v3.0.1 are the travis definitions

### CODE REVIEW

File: NamedTuples/versions/3.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5.0-
```
Comment (tkelman) [inline NamedTuples/versions/3.0.1/requires]: do you still need to support prereleases of 0.5? if not, drop the trailing dash

Comment (mdcfrancis) [inline NamedTuples/versions/3.0.1/requires]: I have dropped, this will require a re-tag to 3.0.2, correct ?

Comment (tkelman) [inline NamedTuples/versions/3.0.1/requires]: or you can edit this file in place in your metadata fork. it's not critical, can wait until next tag. up to you.

**Comment by mdcfrancis** [comment]
Updated version here and also in the NamedTuples package. Will be included in the next tag automatically.

**Comment by mdcfrancis** [comment]
thanks

**Pull request merged**